esp/templates/program/modules/teachercheckinmodule/missingteachers.html

```diff
@@ -155,16 +155,22 @@ <h1>Missing Teachers for {{ program.niceName }}</h1>
 {% ifchanged section.all_arrived %}
     {% if section.all_arrived == True %}
     <tr>
-        <th colspan="6" style="border-top: 4px solid #ccc;">
+        <th colspan="5" style="border-top: 4px solid #ccc;">
             Sections with all teachers checked in
         </th>
     </tr>
     {% endif %}
 {% endifchanged %}
 <tr class="section-first-row" data-sec-id="{{ section.emailcode }}">
-    <td rowspan="{% if section.teachers %}{{ section.teachers|length }}{% else %}1{% endif %}" class="section-code">
-        {{ section.parent_class.emailcode }}: {{ section.name }}
+    <td class="section-detail-header" title="Click to toggle class details" colspan="5">
+        {{ section }}
+        (<a href="{{section.get_absolute_url|urlencode}}">manage</a> | <a href="{{section.get_edit_absolute_url|urlencode}}">edit</a>)
     </td>
+</tr>
+<tr>
+    <!-- <td rowspan="{% if section.teachers %}{{ section.teachers|length }}{% else %}1{% endif %}" class="section-code"> -->
+        <!-- {{ section.parent_class.emailcode }}: {{ section.title }} -->
+    <!-- </td> -->
```

Comment:
Can you delete this commented out code ?

Reply:
Whoops, good catch!